City-of-Helsinki/drupal-hdbt
# UHF-29 Component library

How to test: 
- Clone hel.fi environment (https://github.com/City-of-Helsinki/drupal-helfi) and run `make new`
- Go to theme directory `cd public/themes/custom/hdbt`
   - and switch to this branch `git fetch && git checkout --track origin/UHF-29_component_library`.
   - Compile the theme by running `nvm use && npm i && npm run build`
- Enable HDBT Component library module `drush en -y hdbt_component_library && drush cr`
- Log in to https://helfi.docker.sh and go to https://helfi.docker.sh/admin/appearance/hdbt/component-library
- Check that the component library looks some-what like the image below
![image](https://user-images.githubusercontent.com/1712902/98810294-ad045980-2427-11eb-94ff-c81c68094dbd.png)
- Check that when editing `./templates/component_library/component-library--hdbt.html.twig` the changes appear to the component library
